src/App.js

```diff
@@ -1,16 +1,42 @@
 import React from 'react';
 import './App.css';
+import { useState } from 'react';
+import ChatLog from './components/ChatLog';
 import chatMessages from './data/messages.json';
 
 const App = () => {
+  
+  const [messageData,setMessageData] = useState(chatMessages)
+  
+  const likeMessage = (id) => {
+    setMessageData(messageData =>
+    messageData.map(message => {
+      if (message.id === id) {
+        const button = document.querySelectorAll('button.like')[id-1]
```

Comment:
We should avoid directly manipulating the DOM in this way when working in React. React is essentially purpose built to update the DOM when responding to state changes.